explainaboard/processors/processor.py

```diff
@@ -356,6 +356,13 @@ def process(self, metadata: dict, sys_output: List[dict]) -> SysOutputInfo:
         active_features = self._complete_features(
             sys_info, sys_output, statistics=statistics
```

Comment:
it seems that we have multiple types of [`statistics`](https://github.com/neulab/ExplainaBoard/pull/162/commits/d863afd2e416a038a2cf4800cc9912f727c44c4d), do we want to make the name of this `statistics` more clear?

Reply:
I think that's fixed in the other PR, which @OscarWang114 will still need to merge into this one.

Reply:
Thanks. Yes, I will merge them now. Also, this is still a draft so just let me know if the high-level direction sounds good, I will convert it to a formal one once the details are properly implemented.

Reply:
thanks! it looks good and I will be ready whenever needed to be reviewed.

Reply:
Thanks! It's ready now.

Reply:
Thanks will do it now.